scripts/01_IAvH_test/area_layers.yml

```diff
@@ -0,0 +1,28 @@
+script: area_layers.R
+description: "This sample script shows how it works."
+external_link: https://github.com/GEO-BON/biab-2.0

```

Comment:
Remove if no external link relevant. Comment valid for all other yml files too.
Add references and DOI attributes if relevant for script.